src/panels/config/integrations/integration-panels/zha/zha-network-visualization-page.ts

```diff
@@ -100,11 +100,9 @@ export class ZHANetworkVisualizationPage extends LitElement {
       if (ieee) {
         const device = this._devices.get(ieee);
         if (device) {
-          navigate(
-            this,
-            `/config/devices/device/${device.device_reg_id}`,
-            false
-          );
+          navigate(`/config/devices/device/${device.device_reg_id}`, {
+            replace: true,
```

Comment:
This was false?

Reply:
oh... my bad, there might be others as well with that, will do a scan and adjust

Reply:
Looks like that was the only one